walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -47,22 +48,23 @@
   }
 
   private Collection<TimeRange> findFreeTimes(ArrayList<TimeRange> eventTimes, long duration) {
+    eventTimes.add(TimeRange.fromStartDuration(TimeRange.END_OF_DAY + 1, 0));
```

Comment:
This is very smart to have this dummy event! One question, for the last free time, should it be inclusive=true or false? Currently it will always be false for all free times.

Reply:
It should be inclusive=true, which is why the dummy event ends one minute after the end of the day (as such, it is equivalent to inclusive=true until the end of the day).

Reply:
Thanks for the explanation. I looked at TimeRange.java and learned that each point is in the unit of minutes.